basiclti/tsugi-util/src/java/org/tsugi/basiclti/BasicLTIUtil.java

```diff
@@ -553,14 +553,18 @@ public static String postLaunchHTML(
 		}
 
 		// Paint the submit button
-		text.append("<input type=\"submit\" value=\"");
-		text.append(htmlspecialchars(launchtext));
-		text.append("\">\n");
-
 		if ( debug ) {
+			text.append("<input type=\"submit\" value=\"");
+			text.append(htmlspecialchars(launchtext));
+			text.append("\">\n");
+
 			text.append(" <input type=\"Submit\" value=\"Show Launch Data\" onclick=\"document.getElementById('ltiLaunchDebug_");
 			text.append(submit_uuid);
 			text.append("').style.display = 'block';return false;\">\n");
+		} else {
+			text.append("<input type=\"submit\" value=\" style=\"display: none\"");
```

Comment:
This renders as:

    <input type="submit" value=" style="display: none"Press to continue to external tool.">

but it should be:

    <input type="submit" style="display: none" value="Press to continue to external tool.">